INMOOV-Head-Control/StandardServo.cpp

```diff
@@ -2,87 +2,87 @@
   m_reverseDirection = reverseDirection;                    //Is the rotaion of the servo diffrent to the roation of the part?
 
   m_minRotation = mapf((minRotation / m_outputGearRatio),    //PWM time minimum for part to rotate full range
-                      0, 
-                      m_degreeRange, 
-                      m_minPWMTime, 
-                      m_maxPWMTime);
-                      
+                       0,
+                       m_degreeRange,
+                       m_minPWMTime,
+                       m_maxPWMTime);
+
   m_maxRotation = mapf((maxRotation / m_outputGearRatio),    //PWM time maximum for part to rotate full range
-                      0, 
-                      m_degreeRange, 
-                      m_minPWMTime, 
-                      m_maxPWMTime);
+                       0,
+                       m_degreeRange,
+                       m_minPWMTime,
+                       m_maxPWMTime);
 
   m_startingRotation = mapf((startingRotation / m_outputGearRatio),  //PWM time to set when the .attach() function is called for home position
-                            0, 
-                            m_rotationRangeDegrees,  
-                            m_minRotation, 
+                            0,
+                            m_rotationRangeDegrees,
+                            m_minRotation,
                             m_maxRotation);
 
   m_currentRotation = startingRotation;                     //Current PWM time of the servo
 }
 
 float StandardServo::mapf(float x, float in_min, float in_max, float out_min, float out_max) {
-            return (x - in_min) * (out_max - out_min) / (in_max - in_min) + out_min;
+  return (x - in_min) * (out_max - out_min) / (in_max - in_min) + out_min;
 }
 
-float StandardServo::degreesToPWM(float degreesToMove){
+float StandardServo::degreesToPWM(float degreesToMove) {
   return (mapf((degreesToMove / m_outputGearRatio),
-                0, 
-                m_rotationRangeDegrees, 
-                m_minRotation,                     
-                m_maxRotation) - m_minRotation);            //Converts change in degrees to change PWM time
+               0,
+               m_rotationRangeDegrees,
+               m_minRotation,
+               m_maxRotation) - m_minRotation);            //Converts change in degrees to change PWM time
 }
 
-bool StandardServo::moveToTargetMicroseconds(float target){   
+bool StandardServo::moveToTargetMicroseconds(float target) {
   bool validMove;
-  if((target > m_maxRotation) || (target < m_minRotation)){ //Check if target rotation is in bounds
+  if ((target > m_maxRotation) || (target < m_minRotation)) { //Check if target rotation is in bounds
     validMove = false;
-  }else{
+  } else {
     validMove = true;
   }
 
   target = constrain(target, m_minRotation, m_maxRotation); //Constain target roation to bounds
 
-  if(m_reverseDirection){
+  if (m_reverseDirection) {
     m_servo.writeMicroseconds(int(mapf(target,
```

Comment:
I believe we can just use the normal Arduino map() function, as we will map it to a int anyways. Check if the outcome value will be the same.

Reply:
The orginal reason for adding mapf as to address a drift issue when using the moveRelative() function. I would say we should leave it in unless we remove the moveRelative function completely.